Sources/MintKit/ProcessInfoExtensions.swift

```diff
@@ -0,0 +1,17 @@
+import Foundation
+
+extension ProcessInfo {
+    /// Returns a `String` representing the machine hardware name or nil if there was an error invoking `uname(_:)` or decoding the response.
+    ///
+    /// Return value is the equivalent to running `$ uname -m` in shell.
+    var machineHardwareName: String? {
```

Comment:
I wonder if I am being too strict here? Could this be non-optional and either just force unwrap or return an empty string in the (unlikely) event of a failure?